examples/tracing/templates/index.html

```diff
@@ -1,4 +1,6 @@
-<script src="https://browser.sentry-cdn.com/5.4.1/bundle.js" crossorigin="anonymous"></script>
+<meta name="sentry-trace" content="{{ sentry_trace }}" />
```

Comment:
This line shouldn't be here as it's being included via `{{ sentry_trace }}` interpolation itself.

Reply:
#1434